src/include/grpc_transcoding/path_matcher.h

```diff
@@ -458,9 +455,11 @@ Method PathMatcher<Method>::Lookup(
   MethodData* method_data = reinterpret_cast<MethodData*>(lookup_result.data);
   if (variable_bindings != nullptr) {
     variable_bindings->clear();
-    ExtractBindingsFromPath(method_data->variables, parts, variable_bindings,
-                            fully_decode_reserved_expansion_ || always_decode_,
-                            always_decode_);
+    ExtractBindingsFromPath(
+        method_data->variables, parts, variable_bindings,
+        unescape_spec_ == UrlUnescapeSpec::kAllCharacters ||
```

Comment:
We should not use two arguments to control one aspect.   We should just one one argument.  Same as UrlUnescape function.

Reply:
Passed `UrlUnescapeSpec` all the way through.